base/loading.jl

```diff
@@ -80,9 +80,19 @@ else
     end
 end
 
+function try_path(prefix::ByteString, base::ByteString, name::ByteString)
+    path = joinpath(prefix, name)
+    isfile_casesensitive(path) && return abspath(path)
+    path = joinpath(prefix, base, "src", name)
+    isfile_casesensitive(path) && return abspath(path)
+    path = joinpath(prefix, name, "src", name)
+    isfile_casesensitive(path) && return abspath(path)
+    return nothing
+end
+
 # `wd` is a working directory to search. defaults to current working directory.
 # if `wd === nothing`, no extra path is searched.
-function find_in_path(name::AbstractString, wd = pwd())
+function find_in_path(name::ByteString, wd = pwd())
```

Comment:
this caused https://github.com/JuliaLang/PkgDev.jl/issues/38#issuecomment-213241146 and needs a converting fallback